handbook/expectations.md

```diff
@@ -1,11 +1,65 @@
+
+:::{admonition} Quick Link
+:class: info
+[Our Code of Conduct](../CODEOFCONDUCT)
+:::
+
+## Be a GitHub Guru
+
+Everything related to Xdev should be done *on GitHub*, if at all possible!  GitHub is *where we work.*  We use GitHub to share with the Team *what we plan to do* by using GitHub Issues and GitHub Projects.  We use GitHub to share with the Team *what we are currently working on* using GitHub Pull Requests.  We engage with the entire Team using GitHub Team Discussions.  As a member of the Xdev Team, or as a collaborator working with the Xdev Team, **you are expected to interact with the team *as much as possible* on GitHub, following the practices describing in the [How we use GitHub section](github) of this handbook.**
+
+Unfortunately, not everything can be done on GitHub.  We understand that, and you should conduct our [Communication Guide](communicating) for how to use other platforms and forms of communication that *are not* GitHub.
+
+:::{admonition} Quick Links
+:class: info
+[How we use GitHub](github) <br>
+[Communication Guide](communicating)
 :::
 
-As a member of Xdev, we expect you to...
+## Meetings are for engagement
 
-Follow the [Code of Conduct](../CODEOFCONDUCT).
+The Xdev Team and Xdev Stakeholders (see [Xdev Governance](../about/xdev#governance)) meet on a regular basis.  Core team members meet on a weekly basis for *synchronous* discussions related to our projects and other team activity.  The broader stakeholder community meets with the core team on a less frequent basis (TBD) to discuss past accomplishments, the current status of the team, and future plans.  **If you are an Xdev core team member, you are expected to attend all of these meetings, barring vacation, conflicting high-priority meetings, etc.**  If you are an Xdev Stakeholder, we hope you will join us at these meetings when you can attend.
+
+:::{admonition} Quick Link
+:class: info
+[Xdev Meetings](meetings)
+:::
+
+## Be honest about how much you can do
+
+We know that nobody in Xdev can contribute all of their time to Xdev work.  Even core team members have other duties and commitments to other things, ranging from the basic requirements of work (e.g., annual performance assessments, required training) to commitments to externally funded projects.  To that end, **core team members are expected to clearly communicate to the rest of the team the level of commitment that you expect to be able to deliver over the next week at the Xdev Weekly Meeting.**
+
+:::{admonition} Don't panic!
+:class: tip
+It's okay to say you can't contribute during a week!  Life happens!  Remember that it is more important to simply share with each other your availability and how much other people can expect of you.  Nobody will judge you negatively if you can't contribute 100% in a given week.  (If you are on vacation, we may be jealous, though.)
+:::
+
+## Daily Standups
```

Comment:
I went ahead and made a [discussion thread](https://github.com/orgs/ncar-xdev/teams/all/discussions/1). But doing so made me realize there are a few options we could take (and document) here. Should we have a new thread every day or one megathread? Will GitHub collapse/truncate the old comments and show the most recent?

Reply:
I think the discussion thread your created works great.  I will link to it in the new Handbook.  

I don't actually know all of the features of GitHub team discussions, so we'll have to see how it goes.  If this is too much of a burden, then we'll need to rethink it.  For the moment, though, I link the megathread.

Reply:
Looks like it shows the most recent comment, which I think is ideal.